packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/propertyComponent.css

```diff
@@ -19,5 +19,5 @@
   text-overflow: ellipsis;
   white-space: nowrap;
   font-weight: normal;
-  width: 288px;
+  width: 355px;
```

Comment:
should it be based on the config panel width ?

Reply:
Yes, but since it overflows when we do that we decided to just increase the width instead